### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by tdincer**

### CODE REVIEW

**Review by kabilar — APPROVED** [review]
Thanks @tdincer!  Please update version and changelog.

**Comment by tdincer** [comment]
@kabilar I did and it still complains.

**Comment by tdincer** [comment]
This is a blocker for me as I'm debugging a SciOps workflow. Any help would be appreciated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,15 +2,18 @@
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+# [0.2.1] - 2022-05-13
```
Comment (kabilar) [inline CHANGELOG.md]:
```suggestion
# [0.1.1] - 2022-05-13
```

Comment (kabilar) [reply CHANGELOG.md]: I would consider this a patch.

File: element_facemap/version.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
 """Package metadata."""
-__version__ = "0.1.0"
+__version__ = "0.2.1"
```
Comment (kabilar) [inline element_facemap/version.py]:
```suggestion
__version__ = "0.1.1"
```

**Comment by kabilar** [comment]
> @kabilar I did and it still complains.

Not sure why the check fails but we can merge after incorporating the suggestion above.  Thanks!

**Review by kabilar — APPROVED** [review]

**Pull request merged**